logstreams/src/test/java/io/zeebe/logstreams/util/SyncLogStream.java

```diff
@@ -73,4 +63,14 @@ public void setCommitPosition(final long position) {
   public LogStreamReader newLogStreamReader() {
     return logStream.newLogStreamReader().join();
   }
+
+  @Override
+  public LogStreamRecordWriter newLogStreamRecordWriter() {
```

Comment:
Which setting exactly we used now? It seems for me it is lexicographical method ordering ?